ratatui/src/terminal/terminal.rs

```diff
@@ -820,6 +820,13 @@ where
         }
         Ok(())
     }
+
+    /// Alert the user in the manner of the VT100 "Bell" function.
+    /// Depending on the backend, the terminal app, and the user's settings,
+    /// this may play a beep, flash the screen, or do nothing at all.
```

Comment:
Same comment re: wrapping.

Leave a line break between the first line and the rest. For doc comments, the first line shows up in the module summary table etc., and the detail only shows up in the main part.